app/controllers/accounts/personal_keys_controller.rb

```diff
@@ -1,10 +1,12 @@
 module Accounts
   # Lets users generate a new personal key
-  class PersonalKeysController < ReauthnRequiredController
+  class PersonalKeysController < ApplicationController
     include PersonalKeyConcern
+    include ReauthenticationRequiredConcern
 
     before_action :confirm_two_factor_authenticated
     before_action :prompt_for_password_if_pii_locked
+    before_action :confirm_recently_authenticated
```

Comment:
thank you for doing these as explicit includes instead of `included do ...` magic!